websites/B/BeatLeader/presence.ts

```diff
@@ -12,7 +12,8 @@ const presence = new Presence({
 		"https://cdn.rcd.gg/PreMiD/websites/B/BeatLeader/assets/79.png", //Poodleader
 		"https://cdn.rcd.gg/PreMiD/websites/B/BeatLeader/assets/80.png", //GladdePaling
 		"https://cdn.rcd.gg/PreMiD/websites/B/BeatLeader/assets/81.gif", //EarthDay
-		`https://i.imgur.com${"/Kf8bXNc.gif"}`, //Christmas
+		`https://i.imgur.com${"/Kf8bXNc.gif"}`, //Christmas 2022
+		"https://i.imgur.com/4dpoXYa.gif", //Christmas 2023
```

Comment:
Probs need to do your same fix for the gif, as christmas 2022. Don't think the gif resizing has been fixed.